src/pages/gateway/command-reference.md

```diff
@@ -401,6 +401,70 @@ aio api-mesh:source:get -m -s "AEM Assets API" -s "Adobe Target API"
 ]
 ```
 
+## aio api-mesh:source:install
+
+Updates the currently selected workspace's mesh configuration with the selected configuration from the specified source.
+
+<InlineAlert variant="info" slots="text"/>
+
+Sources are prebuilt mesh configuration files that are formatted for a specific combination of handlers. Each source contains a mesh configuration file designed for a specific first or third-party source. Third-parties can submit their sources as a pull request to the [api-mesh-sources](https://github.com/adobe/api-mesh-sources) repo. See [Create a mesh from a source](./create-mesh.md#create-a-mesh-from-a-source) for more information.
+
+### Usage
+
+```bash
+aio api-mesh:source:install "SOURCE_NAME"
+```
+
+To install a specific version of a source, use the following command:
+
+```bash
+aio api-mesh:source:install "SOURCE_NAME"@source_version_number
+```
+
+### Flags
+
+`-v` or `--variable` specifies the values of any variables in the `variables` array in the mesh configuration file for the source.
```

Comment:
You mention an array, but it's not clear to me whether multiple variables should be expressed as a comma separated list, or one -v flag per variable.